File: oa-portal/problems/m60-feed-dedup/starters/main.java
import java.io.*;
import java.util.*;

public class Main {

    /* Each post is one whole line and may contain spaces. */
    static int distinctPosts(String[] posts) {
        // write your code here

        return 0;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine().trim());
        String[] posts = new String[n];
        for (int i = 0; i < n; i++) {
            String line = br.readLine();
            posts[i] = line == null ? "" : line;
        }

        System.out.println(distinctPosts(posts));
    }
}

File: oa-portal/problems/m60-feed-dedup/starters/main.cpp
#include <bits/stdc++.h>
using namespace std;

/* Each post is one whole line and may contain spaces. */
int distinctPosts(const vector<string>& posts) {
    // write your code here

    return 0;
}

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n;
    if (!(cin >> n)) return 0;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');
    vector<string> posts(n);
    for (int i = 0; i < n; i++) getline(cin, posts[i]);

    cout << distinctPosts(posts) << '\n';
    return 0;
}
